src/compressor/QatAccel.h

```diff
@@ -15,15 +15,23 @@
 #ifndef CEPH_QATACCEL_H
 #define CEPH_QATACCEL_H
 
-#include <qatzip.h>
 #include <boost/optional.hpp>
 #include "include/buffer.h"
+#include <memory>
+
+struct QzSession_S; // typedef struct QzSession_S QzSession_T;
+
+struct QzSessionDeleter {
+  void operator() (struct QzSession_S *session);
```

Comment:
s/struct//

Reply:
The `struct` may not be deleted, because `struct QzSession_S` comes from QAT driver libraries in C.